### ISSUE TITLE
metadata group

### ISSUE THREAD

**Issue by macMikey** [macMikey/LCB-missing-manual#2]
I can only find examples of **group** being used in datatype files, not in any widget.

**Comment by bwmilby** [comment]
Since this is just used to group items in the PI (put them on the same row if possible), that is probably why it hasn't been used in public widget code.

**Comment by macMikey** [comment]
Right. I just want to make sure we have it documented correctly. Where it has been used in a file, the contents of the column don't seem to refer to anything else in the file.

**Comment by macMikey** [comment]
https://github.com/macMikey/LCB-missing-manual/wiki/Property-Editors

**Comment by macMikey** [comment]
Need to test, update [this page](https://github.com/macMikey/LCB-missing-manual/wiki/property-metadata#group) and close this issue
